jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java

```diff
@@ -1,34 +1,121 @@
 package io.github.epam.angular.tests.elements.complex;
 
+import io.github.com.pages.sections.RadioButtonSection;
 import io.github.epam.TestsInit;
+import org.hamcrest.Matchers;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 import static io.github.com.StaticSite.angularPage;
-import static io.github.com.pages.AngularPage.basicRadioButtons;
-import static io.github.com.pages.AngularPage.firstBasicRadioButton;
-import static io.github.com.pages.AngularPage.secondBasicRadioButton;
+import static io.github.com.pages.AngularPage.*;
 import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+import static org.apache.tika.mime.MediaType.text;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.matchesRegex;
 import static org.testng.Assert.assertEquals;
 
```

Comment:
There are many unused import statements in the class.
Please format file (delete unused import statements).